index.bs

```diff
@@ -701,14 +739,17 @@ spec:url; type:dfn; text:scheme
 
   A {{PortalActivateEvent}} has an associated <dfn for="PortalActivateEvent">predecessor browsing context</dfn>,
   which is a [=top-level browsing context=] or null, a <dfn for="PortalActivateEvent">successor window</dfn>, which is
-  a {{Window}}, and an <dfn for="PortalActivateEvent">activation promise</dfn>, which is a [=promise=].
+  a {{Window}}, an <dfn for="PortalActivateEvent">activation promise</dfn>, which is a [=promise=], and a
+  <dfn for="PortalActivateEvent">adopted predecessor element</dfn>, which is a <{portal}> element or null.
```

Comment:
The fact that this is an element doesn't appear to be used. A boolean would work just as well. Right?

Reply:
It's used to clear the "just adopted" flag after `portalactivate` dispatch.

Reply:
You're right, my bad.